aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/CopyPartIntegrationTest.java

```diff
@@ -52,22 +48,22 @@
 public class CopyPartIntegrationTest extends S3IntegrationTestBase {
```

Comment:
Can you include a description of the changes to this file in the PR description?